dvc/repo/status.py

```diff
@@ -10,28 +10,32 @@
 logger = logging.getLogger(__name__)
 
 
-def _local_status(self, targets=None, with_deps=False):
+def _joint_status(stages):
     status = {}
 
-    if targets:
-        stages = cat(self.collect(t, with_deps=with_deps) for t in targets)
-    else:
-        stages = self.collect(None, with_deps=with_deps)
-
```

Comment:
I guess you've split these because of deep source, right? Nothing wrong with that, just asking 🙂

Reply:
That's exactly it :)